scenes/ui/screen/world_screen/world_screen.gd

```diff
@@ -44,6 +44,10 @@ func _load_progress_button(resource_generator_id: String) -> void:
 func _add_progress_button(resource_generator: ResourceGenerator) -> ProgressButton:
 	var progress_button_item: ProgressButton = progress_button_scene.instantiate() as ProgressButton
 	progress_button_item.set_resource_generator(resource_generator)
+	if resource_generator.sfx_generate:
```

Comment:
If you implement the previous export's comments, we can have default audio id's in the resource.